tools/build-monaco-template.sh

```
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+CURRENT_PATH=$(pwd)
+SCRIPTPATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)